src/GraphQL.ApiTests/GraphQL.approved.txt

```diff
@@ -1767,6 +1779,14 @@ namespace GraphQL.Types
         public GraphQL.Resolvers.IAsyncEventStreamResolver AsyncSubscriber { get; set; }
         public GraphQL.Resolvers.IEventStreamResolver Subscriber { get; set; }
     }
+    public class FieldFields : System.Collections.Generic.IEnumerable<GraphQL.Types.FieldType>, System.Collections.IEnumerable
```

Comment:
This sounds funny.  How about `FieldList`?

Reply:
At first I named it just `Fields` but we already have this name `GraphQL.Language.AST.Fields`. If I call it `FieldList`, then it will be logical to give the `List` suffix to all such classes. I don't know.